Directory.Build.props

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the MIT license. See License.txt in the project root for full license information. -->
+<Project>
+
+  <!-- This file is imported by all projects at the beginning of the project files -->
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <Configuration Condition="'$(Configuration)' == ''">Debug</Configuration>
+  </PropertyGroup>
+
+  <Import Project="$(MSBuildThisFileDirectory)build\NuGet.props"/>
+  <Import Project="$(MSBuildThisFileDirectory)build\Versions.props"/>
+
+  <PropertyGroup>
+    <RepoRoot>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\'))</RepoRoot>
```

Comment:
>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\')) [](start = 14, length = 64)

This can be just $(MSBuildThisFileDirectory). Turns out it is documented to already include the final backslash.